File: BEE 1079.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        for (int i = 0; i < n; i++){
            float num1 = input.nextFloat();
            float num2 = input.nextFloat();
            float num3 = input.nextFloat();
            float avg = ((num1 * 2) + (num2 * 3) + (num3 * 5)) / (2 + 3 + 5);
            System.out.printf("%.1f\n", avg);
        }
    }
}

File: BEE 1079.cpp
#include <bits/stdc++.h>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(1) << (x)

using namespace std;

int main(){
    int n;
    cin >> n;
    for (int i = 0; i < n; i++){
        float num1, num2, num3;
        cin >> num1 >> num2 >> num3;
        float avg = ((num1 * 2) + (num2 * 3) + (num3 * 5)) / (2 + 3 + 5);
        cout << FIXED_FLOAT(avg) << endl;
    }
    return 0;
}